src/Microsoft.Health.Dicom.Tests.Common/Samples.cs

```diff
@@ -248,6 +248,22 @@ private static DicomDataset GenerateNewDataSetWithInvalidVr()
             return ds;
         }
 
+        public static DicomDataset CreateCancelWorkitemInstanceDataset(string workitemUid, string cancellationReason)
+        {
+            var ds = new DicomDataset
```

Comment:
This dataset should only contain the attributes here: https://dicom.nema.org/medical/dicom/current/output/html/part04.html#table_CC.2.2-1